example_site/src/components/ConfigurationControls.js

```diff
@@ -54,16 +53,36 @@ export default function ConfigurationControls({ isOpen }) {
     shouldShowConfirmCompleteVideoModal,
   } = watchFields;
 
-  const onSubmit = async (formData) => {
-    let configuration = formData;
+  /**
+   * Returns proper configuration settings for SDK Demo
```

Comment:
I would put a little something about the partner in here - it will shed a little light about what the "proper" configuration settings are.